site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -173,4 +196,35 @@
         $("#peer-table").on("click", ".fa-pencil-alt", adminGradeablePeerForm);
         $('#admin-gradeable-peers-form').on("change", ":text", checkSubmitValid);
     });
-</script>
\ No newline at end of file
+    function downloadPeerDataAsCsv(table_id) {
+    // Select rows from table_id
+    var rows = document.querySelectorAll('table#' + table_id + ' tr');
```

Comment:
Try and avoid using `var` when possible using `const` and `let` are safer alternatives in JS, for example, `const rows`

Reply:
corrected!